src/Symfony/Bridge/Twig/Resources/views/Email/notification_body_markdown.html.twig

```diff
@@ -0,0 +1,3 @@
+{% filter markdown_to_html %}
+    {{ raw ? content|raw : content }}
```

Comment:
If the content is some Markdown, we don't want to escape it before Markdown (as HTML in markdown is something allowed).

Reply:
I'm talking abotu the case where `raw=false`. That would apply HTML escaping on the markdown source, before passing it to the `markdown_to_html` filter (because of using a `filter` tag on some output rather than filtering an expression before displaying it)

Reply:
Which means that `raw` should have no effect with Markdown. I'm going to change `markdown()` to take the content as suggested by @lyrixx so that you can use `content()` with `raw` or `markdown()` depending on your needs.

Reply:
well, the way the template is written right now means that `raw` is **required** if you don't want to have some HTML escaping applied and breaking your markdown.

I suggest changing the template to

```
{{ content|markdown_to_html }}
```

This avoids applying some HTML escaping in a place expecting a markdown context.

Reply:
That's already the case, you haven't looked at the latest version of the PR.